Add Meeting 73.md (#135)
ethereum/pm: All Core Devs Meetings/Meeting 73.md

# Ethereum Core Devs Meeting 73 Notes
### Meeting Date/Time: Friday 25 October 2019 at 14:00 UTC
### Meeting Duration: 1.5 hrs scheduled, 1 hour 5 minutes actual
### [GitHub Agenda Page](https://github.com/ethereum/pm/issues/133)
### [Audio/Video of the meeting](https://www.youtube.com/watch?v=zT4TzlXQ6wA)
### Moderator: Hudson Jameson
### Notes: Jim Bennett

----

# Previous Decisions

**DECISION 72.1**: The Ice Age EIP will not be included in Istanbul. This gives enough time to plan for another fork and not delay Istanbul.

**DECISION 72.2**: Bring your objections to the AllCoreDevs in two weeks if anyone is opposing the Geth team defining Eth64 as the fork ID thing and rolling it out from EIP 2124.

**DECISION 72.3**: CatHerders are going to reach out to the Gorley team to make sure that there is clear communication about node upgrades for Istanbul.

> **DECISION 72.4**: David Palm will talk to Wei to see if 2200 should be self-contained.

Reviewer: ```suggestion
**DECISION 72.4**: Wei has confirmed to David and other core developers that 2200 is self-contained.
```
